src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  &_img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+  }
+
+  &__info {
+    margin-top: 40px;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-product-code {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $secondary-text-color;
+      margin-top: 4px;
+      text-align: left;
+    }
+  }
+
+  &__review {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+    }
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    &-title {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-count {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $button-coior;
+    color: #fff;
+    border: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+
+    &:hover {
+      border: 1px solid $button-coior;
+      background-color: #fff;
+      color: $button-coior;
+    }
```

Comment:
The `&:hover` state of the button should not change the `border` property if it's not intended to affect the button's size. Adding a border on hover can cause the button to shift if it did not already have a border of the same width. To prevent this, you can use `outline` instead of `border` or ensure the default state of the button has a transparent border of the same width.